BrightContrastModule.py

```diff
@@ -56,4 +57,10 @@ def getContrast(s,params):
     #RMS contrast
     rms_contrast=np.sqrt(pow(img - img.mean(), 2).sum() / img.size)
     s.addToPrintList("rms_contrast", str(rms_contrast))
+         
+    #TenenGrad contrast
+    tmp = sobel(img)
```

Comment:
looking at this sobel(img)

doesn't this assume that img is a matrix?

Above, if limit_to_mask is true:

```
    if (limit_to_mask):
        img = img[s["img_mask_use"]]
```

isn't img then a vector, as its been indexed by a binary mask matrix, and not a vector?